DashAI/back/preprocessor/base_preprocessor.py

```diff
@@ -0,0 +1,24 @@
+from abc import abstractmethod
+from typing import Final
+
+
+class BasePreprocessor:
```

Comment:
All base class that are configurable objects should extend ConfigObject. Also, as abstract class, it should extend ABC or have ABCMeta as abstract class

```suggestion
from abc import ABCMeta, abstractmethod
from DashAI.back.config_object import ConfigObject


class BasePreprocessor(ConfigObject, metaclass=ABCMeta):
```